_docs-2/getting-started/design.md

```diff
@@ -88,6 +88,9 @@ Multiple Monitors can be run to provide hot-standby support in the face of failu
 forwarding of logs from remote hosts to the Monitor, only one Monitor process should be active
 at one time. Leader election will be performed internally to choose the active Monitor.
 
+### Audit
+Accumulo has a robust and fully configurable audit logging service that logs most table actions, both successful and failed attempts.  Audit logs are managed through the slf4j named logger `org.apache.accumulo.audit`.  The logger can be added to an existing logging framework or the sample file  `conf/log4j-service.properties` can be used to provide basic audit logging.
+
```

Comment:
I think this section looks good, but I don't know if this is the right location for it. Do we have a better location, that talks about logging more generally, where it would be more appropriate?

Reply:
I moved it to in-depth install under the logging section, let me know if that works

Reply:
Chris, also, sorry about the poorly formatted git commit comments across my branches.  I get it now...won't happen again :-)

Reply:
No worries. It's something everyone does poorly at some time or another. I like to joke that the reason I try to suggest people improve their commit log messages is so that I will continue to hold the record for the longest and worst commit message in Accumulo's git history: https://github.com/apache/accumulo/commit/8851c3578e5b12281a47fdb0b0f7c43516a544d6
(it's pretty bad)